### PR TITLE
Add common code

### PR DESCRIPTION
**Pull request by saledouble**
Related to https://github.com/ponder-lab/Logging-Level-Evolution-Plugin/issues/18

### CODE REVIEW

File: edu.cuny.citytech.refactoring.common.core/.classpath [hunk 1]
```diff
@@ -2,6 +2,6 @@
 <classpath>
 	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
 	<classpathentry kind="src" path="src"/>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
```
Comment (khatchad) [inline edu.cuny.citytech.refactoring.common.core/.classpath]: Why is this needed?

**Comment by saledouble** [comment]
Removed useless metadata.
